fe1-web/src/core/network/jsonrpc/messages/MessageRegistry.ts

```diff
@@ -108,7 +113,17 @@ export class MessageRegistry {
   handleMessage(msg: ProcessableMessage): boolean {
     const data = msg.messageData;
     const messageEntry = this.getEntry(data);
-    return messageEntry.handle!(msg);
+
+    const success = messageEntry.handle!(msg);
+
+    if (success) {
+      // execute all after processing handlers
+      for (const handler of this.afterProcessingHandlers) {
+        handler(msg);
+      }
+    }
+
```

Comment:
this feels very ad-hoc.

It's true that we could image a pre-processing and post-processing of `ProcessableMessage`, but I could imagine that we may have scenarios where a given pre-processing or post-processing applies to one message but not the other.

Taking a step back, do we really need something like this ? I'd opt for YAGNI 😅

Reply:
I will re-implement it using redux 👍